frontmatter/cover.markdown

```diff
@@ -1 +1,3 @@
+[TODO: Book Team Lead - Add cover]
+
 ![cover](images/cover.png)
```

Comment:
This file never needs changing. the cover is always called cover, the TODO should be in publish.yaml to check the newest one is uploaded. I do that.

Reply:
I'll update it & remove the instructions.

I can add a TODO to yaml if you want!